### ISSUE TITLE
How can I add custom tags that would be similar to "inline" and "static" to some functions?

### ISSUE THREAD

**Issue by randolf** [doxygen/doxygen#9702]
Is there a way to add tags that would be similar to "inline" and "static" that appear to the right-hand side of functions, methods, etc.?  If so, how can this be done?  Thanks.

**Comment by albert-github** [comment]
I assume you are talking about the words `inline` and `static` as shown here on the right:

![image](https://user-images.githubusercontent.com/5223533/204237395-ff9e4c3c-1eb6-40a1-9c0c-0faf675b00b5.png)

and in the PDF between the square brackets:

![image](https://user-images.githubusercontent.com/5223533/204237885-6453a72f-6386-4e3d-ae0c-3d59666c7583.png)


- there are currently no facilities to add custom labels.
- what is the use case for this?

**Comment by randolf** [comment]
Thanks.  If I want to indicate that a method is or will be deprecated in the future, or that the method is not part of the standard feature set of another library it's based on, then the ability to add such tags would be helpful to people reading the documentation.

**Comment by randolf** [comment]
Yes, that's exactly what I'm referring to -- thanks.

If I want to indicate that a method is (or will be) deprecated in the future, or that the method is not part of the standard feature set of another library that my class is based on, then the ability to add such tags could be helpful to people reading the documentation.

**Comment by albert-github** [comment]
For deprecated doxygen has the `\deprecated` though the results are a bit different from e.g. `static` etc.

Having e.g.
```
/// \file



/// the fie
/// \deprecated The text
void fie1();
```

which results in :

![image](https://user-images.githubusercontent.com/5223533/206895085-ed115a0f-8d98-4e87-ae8f-5eb2723facb0.png)

Creating a flexible way analogous to things like `static` would be an enhancement.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #9730

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.6.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
